src/distrs/binom.jl

```diff
@@ -16,4 +16,12 @@ import .RFunctions:
 binompdf(n::Real, p::Real, k::Real) = exp(binomlogpdf(n, p, k))
 
 # logpdf for numbers with generic types
-binomlogpdf(n::Real, p::Real, k::Real) = -log1p(n) - logbeta(n - k + 1, k + 1) + k * log(p) + (n - k) * log1p(-p)
+function binomlogpdf(n::Real, p::Real, k::Real)
+    if isinteger(k) & (zero(k) <= k <= n)
+        x = loggamma(n + 1) - loggamma(k + 1) - loggamma(n - k + 1) + xlogy(k, p) + xlogy(n - k, 1 - p)
```

Comment:
Break lines at 92 chars:
```suggestion
        x = loggamma(n + 1) - loggamma(k + 1) - loggamma(n - k + 1) +
            xlogy(k, p) + xlogy(n - k, 1 - p)
```